task_01/index.html

```diff
@@ -0,0 +1,119 @@
+<html>
+  <head>
+    <style>
+      #container{
+        font-family:monospace;
+        font-weight:bold;
+      }
+      table, th, td {
+        border: 1px solid;
+        text-align: center;
+        font-size: 140px;
+        width: 33.33%;
+        height: 33.33%;
+        -moz-user-select: none;
+        -webkit-user-select: none;
+        -ms-user-select:none;
+        user-select:none;
+        -o-user-select:none;"
+       }
+      td:hover {
+        background-color: #f5f5f5;
+      }
+      table {
+        width: 100%;
+        height: 100%;
+      }
+    </style>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+  </head>
+
+  <body>
+
+    <div id="container"></div>
+
+    <script>
+      $("#container").text("Krestiki Noliki");
+      var stateMatrix = [[" "," "," "],[" "," "," "],[" "," "," "]];
+      function resetGame() {
+        stateMatrix = [[" "," "," "],[" "," "," "],[" "," "," "]];
+
+      }
+      function doMove() {
+        var i,j;
+        [i, j] = this.id.split('_').map(function(ind){ return +ind });

```

Comment:
better to store element's position at data- attributes
check here https://developer.mozilla.org/ru/docs/Web/Guide/HTML/Using_data_attributes